topics/variant-analysis/tutorials/trio-analysis/workflows/main_workflow.ga

```diff
@@ -71,53 +67,55 @@
                 }
             ],
             "position": {
-                "left": 4.2333221435546875,
-                "top": 182.3499755859375
+                "left": 4.234387908888152,
+                "top": 182.34374789255511
             },
             "post_job_actions": {},
-            "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/ega_download_client/pyega3/4.0.0+galaxy0",
+            "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/ega_download_client/pyega3/4.0.5+galaxy2",
             "tool_shed_repository": {
-                "changeset_revision": "1162bfd9f782",
+                "changeset_revision": "9564758e8638",
                 "name": "ega_download_client",
                 "owner": "iuc",
                 "tool_shed": "toolshed.g2.bx.psu.edu"
             },
-            "tool_state": "{\"action\": {\"action_type\": \"list_dataset_files\", \"__current_case__\": 0, \"dataset_id\": \"EGAD00001008392\"}, \"output_log\": \"false\", \"__page__\": null, \"__rerun_remap_job_id__\": null}",
-            "tool_version": "4.0.0+galaxy0",
+            "tool_state": "{\"action\": {\"action_type\": \"list_dataset_files\", \"__current_case__\": 0, \"dataset_id\": \"EGAD00001008392\"}, \"output_log\": false, \"__page__\": null, \"__rerun_remap_job_id__\": null}",
+            "tool_version": "4.0.5+galaxy2",
             "type": "tool",
             "uuid": "265bf049-ac9f-4a59-bba3-311c6c818121",
+            "when": null,
             "workflow_outputs": [
                 {
-                    "label": "List of files in EGAD00001008392",
+                    "label": null,
```

Comment:
Please make sure that "workflow_outputs" for every step have labels.
The "best practice" wizard in the WF editor should also warn about this.

Reply:
We don't current mandate that. Are the workflow output labels so important that the GTN should start mandating it?

Personally I find renaming every single output file a bit frustrating to do when they most have already ok names, but maybe that's just me being difficult. And especially in training workflows we often keep every output by default which planemo also says is bad but, isn't really as relevant for the training case.

Reply:
@JasperO98 whenever you have time, if we could trouble you to select the important outputs from the workflow and label those, that would be helpful. We will begin mandating it.

Reply:
@JasperO98 whenever you have time, if we could trouble you to select the important outputs from the workflow and label those, that would be helpful.

Reply:
@JasperO98 whenever you have time, if we could trouble you to select the important outputs from the workflow and label those, that would be helpful. We will begin mandating it.

Reply:
(sorry for the triple posting. gh was being unhelpful.)

Reply:
The workflow_output labels should be added already.